README.md

```diff
@@ -102,8 +103,8 @@ class SchedulerTypes(LookupEnum):
     ConstantLR = torch.optim.lr_scheduler.ConstantLR
 
 class StepLRConfig(BaseModel):
-    step_size: conint(ge=1)
-    gamma: confloat(ge=0.0)
+    step_size: Annotated[int, Field(strict=True, gt=1)]
+    gamma: Annotated[float, Field(strict=True, gt=0.0)]
```

Comment:
here, and also in other cases  `ge` was replaced by `gt`. Leaving it as a comment, that we have to fix it.

Reply:
Oops, good catch! I fixed this in fb0aea5fe1fdb04dae9f6299ed2b5f39b4a3be17. Maybe we should consider using only `ge` for the sake of consistency. Type aliases might also help to avoid these kind of mistakes in the future, see  my comment here: https://github.com/Modalities/modalities/issues/57#issuecomment-1961687570